feed.php

```diff
@@ -28,7 +28,7 @@ class FeedPlugin extends Plugin
     /**
      * @var array
      */
-    protected $valid_types = array('rss','atom');
+    protected $valid_types = array('rss','atom','json');
```

Comment:
I removed this from the standard list on purpose.  It's added back if the configuration has the json feed support enabled.